src/language/class-modifiers-for-apis.md

```diff
@@ -232,7 +232,7 @@ In return, you have the fewest restrictions as the class maintainer.
 You can add new methods, turn constructors into factory constructors, etc.
 without worrying about breaking any downstream users.
 
-## The `sealed` modifer
+## The `sealed` modifier
```

Comment:
Wow, kind of embarrassed I missed this so long. Thanks for noticing and fixing this!

Just in case people linked to this header, can we add an empty anchor for the old header so their link still works?

```suggestion
<a id="the-sealed-modifer"></a>
## The `sealed` modifier
```